README.md

```diff
@@ -27,7 +27,24 @@ The diagram below illustrates high-level design of each component as well as bas
 
 ## Usage
 
-Before you can run the Miden node, you'll need to make sure you have Rust [installed](https://www.rust-lang.org/tools/install). Miden node v0.1 requires Rust version **1.73** or later.
+### Prerequisites
+
+Before you can build and run the Miden node or any of its components, you'll need to make sure you have Rust [installed](https://www.rust-lang.org/tools/install). Miden node v0.1 requires Rust version **1.73** or later.
+
+Also make sure all of this libraries are installed:
+
+Ubuntu 22.04:
+
+```sh
+sudo apt install gcc llvm clang bindgen pkg-config libssl-dev libsqlite3-dev
```

Comment:
I believe that it's too cluttered to list all needed packages considering that most of them are installed by default on machines or are installed by rustlings and others on setup.

Reply:
Installing rust should suffice.

Reply:
I would probably get rid of MacOS example here, but would keep Ubuntu. I'd also do the following:

1. Get rid of the `Prerequisites` subheader.
2. Change the intro "Also make sure all of this libraries are installed:" sentence to something like:

"Depending on the platform, you may need to install additional libraries. For example, on Ubuntu 22.04 the following command ensures that all required libraries are installed."

Reply:
@phklive 
> I believe that it's too cluttered to list all needed packages considering that most of them are installed by default on machines or are installed by rustlings and others on setup.

I think, we should provide dependency list at least for those users, who unfamiliar to rust. Because googling compiler errors and installing needed dependencies, might be challenging for them.

I personally prefer, when I do all according to instructions and it just works instead of spending hours trying to figure out, what to do in order to get rid of compiling errors. :)